packages/juggler/package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "@loopback/juggler",
+  "version": "4.0.0-alpha.1",
+  "description": "Juggler for LoopBack.next",
+  "main": "index",
+  "scripts": {
+    "acceptance": "mocha --opts ../../test/mocha.opts 'test/acceptance/**/*.ts'",
+    "build": "npm run build:lib && npm run build:lib6",
+    "build:lib": "tsc --target es2017 --outDir lib",
+    "build:lib6": "tsc --target es2015 --outDir lib6",
```

Comment:
Please use the new build setup from #300. You will also need to add `packages/juggler/index.ts` file.